### ISSUE TITLE
Sampled Action got all "NaN"

### ISSUE THREAD

**Issue by BrunoFANG1** [rail-berkeley/hil-serl#6]
When I ran train_bc.py, I turn "tanh_squash_distribution" to "True" and my agent's sampled actions got all "NaN" after one gradient update. I want to know if there are any numerical issues here. Here is a code snippet of the bc.py I used. Other configurations are all default.

```    @classmethod
    def create(
        cls,
        rng: PRNGKey,
        observations: FrozenDict,
        actions: jnp.ndarray,
        # Model architecture
        encoder_type: str = "resnet-pretrained",
        image_keys: Iterable[str] = ("image",),
        use_proprio: bool = True, ### This is where I modified
        network_kwargs: dict = {
            "hidden_dims": [256, 256],
        },
        policy_kwargs: dict = {
            "tanh_squash_distribution": False,
        },
        # Optimizer
        learning_rate: float = 3e-4,
        augmentation_function: Optional[callable] = None,
    ):```

**Comment by charlesxu0124** [comment]
Thanks for raising the issue. You are right that it is due to numerical stability for actions close to ±1 when using the tanh_squash_distribution. I've implemented an action clipping function to resolve the issue.
